lib/ansible/modules/cloud/azure/azure_rm_functionapp.py

```diff
@@ -38,8 +38,8 @@
         description:
             - App service plan.
             - It can be name of existing app service plan in same resource group as function app.
-            - "It can be resource id of existing app service plan. eg.,
-              /subscriptions/<subs_id>/resourceGroups/<resource_group>/providers/Microsoft.Web/serverFarms/<plan_name>"
+            - It can be resource id of existing app service plan.
+            - Reource id, eg. C(/subscriptions/<subs_id>/resourceGroups/<resource_group>/providers/Microsoft.Web/serverFarms/<plan_name>).
```

Comment:
should this enclosed with C()?

Reply:
It's my mistake, the “C()” should used to files and option value, I will changed this!  Thank you very much!